certified-connectors/PDFco/apiDefinition.swagger.json

```diff
@@ -0,0 +1,727 @@
+                "credits": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "",
+                  "title": "credits",
+                  "x-ms-visibility": "advanced"
+                },
+                "duration": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "",
+                  "title": "duration",
+                  "x-ms-visibility": "advanced"
+                },
+                "remainingCredits": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "",
+                  "title": "remainingCredits",
+                  "x-ms-visibility": "advanced"
+                }
+              }
+            }
+          }
+        },
+        "summary": "URL to PDF",
+        "description": "Create PDF from URL",
+        "operationId": "URL_to_PDF",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "url": {
+                  "type": "string",
+                  "description": "Link to input HTML file or website page to be converted. You can pass link to file from Google Drive, Dropbox or another online file service that can generate shareable links.",
+                  "title": "url",
+                  "x-ms-visibility": "important"
+                },
+                "margins": {
+                  "type": "string",
+                  "description": "set to css style margins like 10px, 5mm, 5in for all sides or 5px 5px 5px 5px (the order of margins is top, right, bottom, left).",
+                  "title": "margins",
+                  "x-ms-visibility": "advanced",
+                  "default": "5mm"
+                },
+                "paperSize": {
+                  "type": "string",
+                  "description": "Letter is set by default. Can be Letter, Legal, Tabloid, Ledger, A0, A1, A2, A3, A4, A5, A6 or a custom size. Custom size can be set in px (pixels), mm or in (inches) with width and height separated by space like this: 200 300, 200px 300px, 200mm 300mm, 20cm 30cm or 6in 8in.",
+                  "title": "paperSize",
+                  "default": "Letter",
+                  "x-ms-visibility": "advanced"
+                },
+                "orientation": {
+                  "type": "string",
+                  "description": "Output page orientation",
+                  "title": "orientation",
+                  "default": "Portrait",
+                  "x-ms-visibility": "advanced",
+                  "enum": ["Portrait", "Landscape"]
+                },
+                "printBackground": {
+                  "type": "boolean",
+                  "description": "enable/disable printing of background",
+                  "title": "printBackground",
```

Comment:
@hiren-byte please look through titles and descriptions and make sure they look like valid sentences, use the links from the above comments.